tests/TestCase/Core/PluginCollectionTest.php

```diff
@@ -177,7 +177,7 @@ function (): void {
 
     public function testCreateException(): void
     {
-        $this->expectException(CakeException::class);
+        $this->expectException(InvalidArgumentException::class);
         $this->expectExceptionMessage('Cannot create a plugin with empty name');
```

Comment:
```suggestion
        $this->expectExceptionMessage('Plugin name cannot be empty.');
```

Missed this was in a test.